sympy/polys/ring_series.py

```diff
@@ -439,10 +431,10 @@ def _series_inversion1(p, x, prec):
         prec = int(prec)
 
     if zm not in p:
-        raise ValueError('no constant term in series')
+        raise ValueError("No constant term in series")
     if _has_constant_term(p - c, x):
-        raise ValueError('p cannot contain a constant term depending on '
-            'parameters')
+        raise ValueError("p cannot contain a constant term depending on "
+                         "parameters")
```

Comment:
Readability does have subjective elements, but it's not just subjectivity.
General consensus has been leaning towards multiples-of-four indents, not hanging indent. Not just in SymPy, but in the Python community in general, though of course everybody's perception tends to be different due to different peer groups. All I can say that in the discussions I've seen on the topic, more often than not, hanging indents are considered an undesirable option, some people even consider their inclusion in PEP8 a mistake; more importantly, I see little if any protest against that stance. In fact you're the first person I ever met who expressed a strong preference for hanging indents.

A more tangible issue with hanging indents is that if the name changes, you have to touch not just the lines with the name but also the names with the parameters on followup lines.

Also, there's the point to be made that subjective preferences shouldn't lead to changes in the code where it already follows another style.